linode/images/framework_schema_datasource.go

```diff
@@ -14,6 +14,8 @@ var filterConfig = frameworkfilter.Config{
 	"type":       {APIFilterable: true, TypeFunc: frameworkfilter.FilterTypeString},
 	"vendor":     {APIFilterable: true, TypeFunc: frameworkfilter.FilterTypeString},
 
+	// TODO: check if tags become API filterable
```

Comment:
The `tags` is described as api filterable in spec doc. However, it doesn't work currently. Leave a comment here for future check.